docs/content/configuring-npm/package-json.md

```diff
@@ -339,12 +339,14 @@ install into the PATH. npm makes this pretty easy (in fact, it uses this
 feature to install the "npm" executable.)
 
 To use this, supply a `bin` field in your package.json which is a map of
-command name to local file name. When this package is installed
-globally, that file will be linked where global bins go so it is
-available to run by name.  When this package is installed as a
-dependency in another package, the file will be linked where it will be
-available to that package either directly by `npm exec` or by name in other
-scripts when invoking them via `npm run-script`.
+command name to local file name. When this package is installed globally,
+that file will be either linked inside the global bins directory or
+a cmd (Windows Command File) will be created which executes the specified 
```

Comment:
Linking inside the global bins directory is also the behavior in posix systems such as linux and os/x